src/java/com/puppetlabs/ssl_utils/SSLUtils.java

```diff
@@ -277,13 +277,17 @@ public static X509Certificate signCertificate(String issuerDn,
      * & public keys, create a new certificate revocation list (CRL).
      *
      * The CRL will have an AuthorityKeyIdentifier extension and CRLNumber
-     * extension set to 0.
+     * extension, set to 0 unless otherwise specified.
      *
      * @param issuer The certificate authority's identifier
      * @param issuerPrivateKey The certificate authority's private key
      * @param issuerPublicKey The certificate authority's public key
      * @param thisUpdate The date after which this CRL becomes valid
      * @param nextUpdate The date by which an updated CRL is expected
+     * @param crlNumber The CRL Number for this CRL, used to identify how new a
+     *                  CRL is compared to another CRL from the same issuer
+     * @param extensions An optional list of extensions to include, in addition
+     *                   to AuthorityKeyIdentifier and CRLNumber.
```

Comment:
If this is optional, would it make sense to have a separate arity without it? I notice we pass `null` most of the time. Don't feel strongly.